plugins/rapid7_insight_agent/help.md

```diff
@@ -447,10 +457,11 @@ Example output:
 
 ## Troubleshooting
   
-*There is no troubleshooting for this plugin.*
+- If the actions `Get Agent Details` and `Get All Agents by IP` return a `next cursor` value, it is an indication that more pages of data are available to be reviewed. In this instance, it is recommended to run the action multiple times and pass the `next cursor` value, recording all agents found.
```

Comment:
maybe since this information is included in the troubleshooting, we don't need to include that second sentence in the actions descriptions?

Reply:
I'd rather keep it in the action description because the change would be potentially disruptive to existing workflows.